DonjinKrawler-Client/src/main/java/donjinkrawler/Game.java

```diff
@@ -247,6 +281,15 @@ public void updateEnemyInfo(String enemyData) {
         }
     }
 
+    public void updateEnemies(List<Enemy> enemies) {
+        for (Enemy enemy : enemies) {
+            AbstractShellInterface updatedEnemy = shells.get(enemy.getID());
+            updatedEnemy.setX(enemy.getX());
```

Comment:
Should probably add a sanity check to see if the updatedEnemy is not null. This might cause NPEs if no check is present.